recipe/0001-skip-osx-test-to-avoid-spurious-segfault.patch

```diff
@@ -1,15 +1,15 @@
 diff --git a/inst/tinytest/test_tiledbarray.R b/inst/tinytest/test_tiledbarray.R
-index b9185e5..4311f95 100644
+index e50d8720..99875684 100644
 --- a/inst/tinytest/test_tiledbarray.R
 +++ b/inst/tinytest/test_tiledbarray.R
 @@ -1,6 +1,10 @@
  library(tinytest)
  library(tiledb)
- 
+
 +# Skip for conda osx-64 build (controlled by Jinja selectors in recipe) due to
 +# spurious test failures
 +exit_file("skip for conda osx-64 build")
```

Comment:
:100:  I was just about to suggest that you make it easier globally by hitting `tests/tinytest.r` and skip there.

Reply:
I don't want to skip all the tests. It was easy enough to update the previous patch to skip `test_tiledbarray.R` on osx-64